### PR TITLE
Update chart

### PR DESCRIPTION
**Pull request by shuo-wu**
https://github.com/longhorn/longhorn/issues/1163

### CODE REVIEW

**Comment by yasker** [comment]
LGTM. Need to test upgrade though.

**Comment by yasker** [comment]
Also need to add instruction on how to use the script. (just run it? but longhorn/longhorn is normally not in the GOPATH)

**Comment by shuo-wu** [comment]
I have removed the script for now since it's not powerful and easy-to-use. I will polish/refactor it later.

File: chart/templates/storageclass.yaml [hunk 1]
```diff
@@ -10,8 +10,8 @@ metadata:
     storageclass.beta.kubernetes.io/is-default-class: "false"
   {{- end }}
 provisioner: driver.longhorn.io
+allowVolumeExpansion: true
 parameters:
   numberOfReplicas: "{{ .Values.persistence.defaultClassReplicaCount }}"
-  staleReplicaTimeout: "30"
+  staleReplicaTimeout: "2880"
```
Comment (yasker) [inline chart/templates/storageclass.yaml:16]: Don't change this for this release. We can add this to part of upgrade to 1.0.

File: chart/templates/deployment-ui.yaml [hunk 2]
```diff
@@ -18,6 +18,7 @@ spec:
       containers:
       - name: longhorn-ui
         image: "{{ .Values.image.longhorn.ui }}:{{ .Values.image.longhorn.uiTag }}"
+        imagePullPolicy: Always
```
Comment (yasker) [inline chart/templates/deployment-ui.yaml:21]: I don't think we have this in the manager?

File: chart/templates/deployment-ui.yaml [hunk 3]
```diff
@@ -28,6 +29,7 @@ spec:
       imagePullSecrets:
       - name: {{ .Values.defaultSettings.registrySecret }}
       {{- end }}
+      serviceAccountName: longhorn-service-account
```
Comment (yasker) [inline chart/templates/deployment-ui.yaml:32]: Seems unnecessary. Need to remove from manager yaml if it's there.

**Pull request merged**